server.js

```diff
@@ -976,26 +985,10 @@ app.get("/network-profiles/:hash(\\w+)", asyncMiddleware(async(req, res) => {
     const pug_data = {
         network,
         http_host: req.protocol + "://" + req.get("host"),
-        matches: await db.collection("matches")
-            .aggregate([
-                { $match: { $or: [{ network1: network.hash }, { network2: network.hash }] } },
-                { $lookup: { localField: "network2", from: "networks", foreignField: "hash", as: "network2" } }, { $unwind: "$network2" },
-                { $lookup: { localField: "network1", from: "networks", foreignField: "hash", as: "network1" } }, { $unwind: "$network1" },
-                { $sort: { _id: -1 } },
-                { $limit: 100 }
-            ]).toArray(),
+        matches: await dbutils.get_matches(db, 100),
```

Comment:
This now is getting all matches instead of the matches that are for the specific network. @bood @rchs819